src/sentry/static/sentry/app/components/charts/components/tooltip.tsx

```diff
@@ -62,22 +83,27 @@ function getFormatter({
     return true;
   };
 
-  return seriesParamsOrParam => {
+  const formatter: EChartOption.Tooltip['formatter'] = seriesParamsOrParam => {
     // If this is a tooltip for the axis, it will include all series for that axis item.
     // In this case seriesParamsOrParam will be of type `Object[]`
     //
     // Otherwise, it will be an `Object`, and is a tooltip for a single item
-    const isAxisItem = Array.isArray(seriesParamsOrParam);
     const axisFormatterOrDefault = formatAxisLabel || defaultFormatAxisLabel;
 
     // Special tooltip if component is a `markPoint`
-    if (!isAxisItem && seriesParamsOrParam.componentType === 'markPoint') {
-      const timestamp = seriesParamsOrParam.data.coord[0];
+    if (
+      !Array.isArray(seriesParamsOrParam) &&
+      // TODO(ts): The EChart types suggest that this can _only_ be `series`,
+      //           but assuming this code is correct (which I have not
+      //           verified) their types may be wrong.
```

Comment:
I'm pretty sure this code is still active and used to generate tooltips for the release marker series.

Reply:
Yeah, something is likely wrong with the typings here in that case unfortunately.